api/src/main/java/org/pmiops/workbench/db/dao/ConceptSetDao.java

```diff
@@ -25,6 +26,8 @@
           .put(Domain.PHYSICALMEASUREMENT, "measurement")
           .build();
 
+  Optional<DbConceptSet> findOneByConceptSetIdAndWorkspaceId(long conceptId, long workspaceId);
```

Comment:
nit: maybe. just `findByConcept...`, to match. the others. teh return type indicates. you're only getting one.

Reply:
Done, also added Test class for ConceptSetDaoTest since it didn't exist.